components/vf-card/vf-card.scss

```diff
@@ -111,6 +111,12 @@
   .vf-card__text {
     --card-text-color: #{set-ui-color(vf-ui-color--black)};
   }
+
+  .vf-card__link {
```

Comment:
Could we slip a link/comment in here? as I'm sure we'll forget why it's here.

Something like:
```
// fixes bug on hover of link and easy variant #1215
```